src/test/java/SteamExecutionsTest.java

```diff
@@ -0,0 +1,68 @@
+import constants.Constants;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.steam.GamePage;
+import org.steam.GameSearchResult;
+import org.steam.HomePage;
+import org.steam.SearchPage;
+import org.testng.annotations.*;
+
+public class SteamExecutionsTest {
+
+    private WebDriver driver;
+    private WebElement webElement;
+
+    @BeforeClass()
+    public void setup(){
+        System.setProperty(Constants.VALUE_DRIVER_SET_PROPERTY, Constants.PATH_SET_PROPERTY);
+        driver = new ChromeDriver();
+        driver.manage().deleteAllCookies();
+        driver.manage().window().maximize();
+        driver.get(Constants.STEAM_URL);
+    }
+
+    @Test()
+    public void searchGameTest() throws InterruptedException {
+        String title = driver.getTitle();
+        System.out.println(title);
+
+        HomePage homePage = new HomePage(driver);
+        homePage.searchAGame("Portal");
+
+        //This is just for the humans can see the result.
+        Thread.sleep(5000);
```

Comment:
nit: If this is a test should have an Assert and compare to expected outputs no need to sleep a thread or read a result

If it's like that due to WIP is fine.

Reply:
Another thing to consider don't know if it's part of the scope but some games are under an "age gate" does this behaves different for those examples? if so need to be considered too.

Reply:
This is a practice of a course that I'm taken with Nearsoft; there is not a specific requirement. I'm learning how improve my skills, feedback, good practice, etc. 

If you wanna give a feedback, it will be a pleasure; you can see other repositories. I know, the last push was 2 years ago.
I appreciate it! Thank you! 
https://github.com/ilse-macias/PageObjectModel/tree/master/PageObjectModel/Guru99POM